packages/lit-html/src/lit-html.ts

```diff
@@ -609,9 +619,28 @@ export interface RenderOptions {
 
 /**
  * Renders a value, usually a lit-html TemplateResult, to the container.
- * @param value
- * @param container
- * @param options
+ *
+ * For example, to render "Hello, Zoe!" to the container `document.body` where
+ * the name "Zoe" is from a variable:
```

Comment:
Maybe to @justinfagnani 's point, this should say something like, 

This example renders the text "Hello, Zoe!" inside a paragraph tag, appending it to the container `document.body`.

Reply:
Thank you! Adding verbatim.